_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation-filter-popin/smart-shopping-campaign-creation-popin-recap.vue

```diff
@@ -139,6 +139,20 @@ export default {
       // TODO also check with PM how it is supposed to be presented
       return this.totalProducts;
     },
+    endDate() {
+      const endDate = new Date(this.newCampaign.endDate).getFullYear();
+      const todayYear = new Date().getFullYear();
+      if (endDate - 10 > todayYear) {
+        return null;
+      }
+      return this.$options.filters.timeConverterToDate(this.newCampaign.endDate);
+    },
+    targetCountry() {
+      if (typeof this.newCampaign.targetCountry !== 'string') {
+        return this.newCampaign.targetCountry[0];
+      }
+      return this.newCampaign.targetCountry;
+    },
```

Comment:
I don't like this workaround, where we try to catch up on an error we made somewhere else. As we're dealing with a single target country, we should never get an array